x-pack/docs/en/security/operator-privileges/index.asciidoc

```diff
@@ -0,0 +1,21 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[operator-privileges]]
+== Operator Privileges
+
+include::operator-privileges-designed-for-note.asciidoc[]
+
+With a typical {es} deployment, people who administer the cluster also operates the cluster at the infrastructure level.
+The RBAC <<authorization>> is effective and reliable for this environment.
+However, in more managed environments, such as {ess-trial}[{ess}], there is a distinction between the operator
+of the cluster infrastructure and the administrator of the cluster.
+The operator privileges feature enhances the {es} security model so that some functionality
+is restricted to be used by infrastructure operators and is not available to cluster administrators,
+even if they log in as a highly privileged user (such as `superuser`).
```

Comment:
Since `superuser` isn't a "user", maybe we want a slightly different phrasing
```suggestion
even if they log in as a highly privileged user (such as the `elastic` user, or another user with the `superuser` role).
```